vignettes/data-combined.Rmd

```diff
@@ -0,0 +1,300 @@
+---
+title: "Working with `DataCombined` class"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Working with `DataCombined` class}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  dpi = 300,
+  comment = "#>",
+  out.width = "100%",
+  fig.height = 6,
+  fig.width = 8
+)
+```
+
+## Introduction
+
+In Modeling and Simulation (M&S) workflows, we often need to work with multiple 
+observed and/or simulated datasets. Further, some of these datasets can be linked 
+together in a group (e.g., the simulated and the correnspoding observed data)
+for analysis and visualization.
+
+The `DataCombined` class in `{ospsuite}` provides a container to store 
+data, to group them, and to transform them further. The class accepts data 
```

Comment:
...and to transform them **(e.g. scale or shift in time)**. ..."
Also here, I would remove _further_

Reply:
Added